README.md

```diff
@@ -1,6 +1,9 @@
-﻿#Microsoft Azure Active Directory Authentication Library (ADAL) for Android
+﻿# Microsoft Azure Active Directory Authentication Library (ADAL) for Android
 ===========
 
+| [Code Samples](https://github.com/azure-samples?utf8=✓&q=active-directory-android) | [Reference Docs](http://javadoc.io/doc/com.microsoft.aad/adal/1.11.0) | [Developer Guide](https://aka.ms/aaddev)
```

Comment:
We considered it, but found this site already solved the problem.  When we do a release, this site generates the docs and hosts it.  For now we'll have to update this link. The site supports "latest" via `.../adal/`, but currently directs to 2.0.4 alpha.